tests/pgshovel/streams/states.py

```diff
@@ -137,63 +152,29 @@ def test_require_different_publisher(message):
 
     require_different_publisher(
         Committed(
-            uuid.uuid1().bytes,  # change the publisher
-            batch_identifier,
+            publisher=uuid.uuid1().bytes,  # change the publisher
+            batch_identifier=batch_identifier,
         ),
+        0,
         message,
     )
 
     with pytest.raises(InvalidPublisher):
         require_different_publisher(
             Committed(
-                message.header.publisher,
-                batch_identifier,
+                publisher=message.header.publisher,
+                batch_identifier=batch_identifier,
             ),
+            0,
             message,
         )
 
 
-def test_stateful_validator():
```

Comment:
I think it makes sense to remove it for now, at least.

It would probably be good to add a test later that is designed more to cover the interface of `StatefulStreamValidator` that isn't tied to the specific details of the transaction state machine (if for no reason other than hygiene) but I think the coverage is plenty good as-is otherwise.